src/styles/stars.scss

```diff
@@ -0,0 +1,24 @@
+@use './variables' as *;
+
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    height: $star-size;
+    width: $star-size;
+    margin-right: $star-margin;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  @for $i from 1 through 5 {
+    &--#{$i} :nth-child(-n + #{$i}) {
+      background-image: $stars-background;
```

Comment:
The `$stars-background` variable usage suggests that you are using a variable for the background image. Ensure that this variable is defined in your `variables.scss` file and holds the correct value for the active star image.